provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/EnrollmentGroupManager.java

```diff
@@ -25,7 +25,7 @@
  * @see <a href="https://docs.microsoft.com/en-us/azure/iot-dps/">Azure IoT Hub Device Provisioning Service</a>
  * @see <a href="https://docs.microsoft.com/en-us/rest/api/iot-dps/deviceenrollmentgroup">Device Enrollment Group</a>
  */
-public class EnrollmentGroupManager
+class EnrollmentGroupManager
```

Comment:
This class and other "Manager" class are just 1 level below our public client, but they shouldn't directly be used